File: CS/1010 - Calculo simples.cs
using System;

class URI
{
	static void Main()
	{
		double total=0;

		for (int i=0; i<2; i++)
		{
			
			string[] tokens = Console.ReadLine().Split();

			int a = int.Parse(tokens[0]);
			int b = int.Parse(tokens[1]);
			double c = double.Parse(tokens[2]);

			total = total + (b * c); 
		}

		Console.WriteLine("VALOR A PAGAR: R$ {0}", total.ToString("0.00"));
	}
}

File: PHP/1010 - calculo simples.php
<?php

$total = 0;

for ($i=0; $i<2; $i++ ){

    $numbers = (string)readline();
    $numbers = explode(" ", $numbers);

    $a = (int)$numbers[1];
    $b = (float)$numbers[2];

    $total += $a*$b;
}

$total = number_format($total, 2, ".", "");
echo "VALOR A PAGAR: R$ $total\n";
?>